### ISSUE TITLE
Move global variable mode into the auxilary array

### ISSUE THREAD

**Issue by JKRT** [JKRT/OMBackend.jl#35]
Currently if equations are supported using a global variable. Merge it with the auxiliary array for booleans integers etc

**Comment by SVAGEN26** [comment]
@JKRT please close: refactoring note about the Mode global from the old generator. That code path is gone.

**Comment by JKRT** [comment]
True, but not gone it is renamed it is the DEGen and work on that is still in progress. This issue is stale though
